k8s-intf/Cargo.toml

```diff
@@ -10,18 +10,25 @@ bolero = ["dep:bolero", "dep:hardware", "dep:net", "dep:lpm", "net/test_buffer",
 
 [dependencies]
 # internal
+hardware = { workspace = true, optional = true, features = ["bolero"] }
+lpm = { workspace = true, optional = true }
+net = { workspace = true, optional = true, features = ["bolero"] }
+tracectl = { workspace = true }
+tracing = { workspace = true }
 
 # external
 bolero = { workspace = true, optional = true }
-hardware = { workspace = true, optional = true, features = [] }
-kube = { workspace = true, features = ["derive"] }
+futures = { workspace = true }
+kube = { workspace = true, features = ["client", "derive", "runtime", "rustls-tls"] }
 k8s-openapi = { workspace = true, features = ["latest", "schemars", "std"] }
-lpm = { workspace = true, optional = true }
-net = { workspace = true, optional = true, features = [] }
+linkme = { workspace = true }
+rustls = { workspace = true, features = ["ring"] }
```

Comment:
I think ring is great, but I also think that the world is generally trending toward [awc-lc-rs](https://github.com/aws/aws-lc-rs) due to extensive formal verification.

You might consider a follow up PR to swap this to aws-lc-rs (it should likely be transparent)

Reply:
Glad to make this change.  I guess that means another test cycle so I can fix up the tokio::spawn stuff.  But I'd like to do it on #1138 instead of this PR.  Is that ok?